modules/data-entry/src/main/java/io/uhndata/cards/DataImportServlet.java

```diff
@@ -583,10 +584,10 @@ private String standardizeValue(final String value, Node question)
                         || !childNode.hasProperty(prop)) {
                         continue;
                     }
-                    if (StringUtils.equals(value, childNode.getProperty(prop).getString())) {
+                    if (Strings.CS.equals(value, childNode.getProperty(prop).getString())) {
                         // We found an exact match for a known option, no need to do any further processing
                         return childNode.getProperty(VALUE_PROPERTY).getString();
-                    } else if (StringUtils.equalsIgnoreCase(value, childNode.getProperty(prop).getString())) {
+                    } else if (Strings.CS.equals(value, childNode.getProperty(prop).getString())) {
```

Comment:
`equalsIgnoreCase` is case insensitive, so it should be migrated to `Strings.CI` instead of `Strings.CS`.
```suggestion
                    } else if (Strings.CI.equals(value, childNode.getProperty(prop).getString())) {
```